### ISSUE TITLE
Using Raw Literals in Update

### ISSUE THREAD

**Issue by arakis** [sqlkata/querybuilder#354]
I'm using the pure Sql builder (for sql server compiler). No Execution.

It seems to be impossible to build such statement:

```
UPDATE table SET column = @column
```

Again, i cannot use classic DB Parameters like "?" or something else, because we use our own SQL execution environment. We need the pure SQL-Statement with a "@var"-Literal.

I tried the following:
```
            var query = new Query("testtable");
            query.AsUpdate(new string[] { "abc" }, new object[] { new UnsafeLiteral("@abc") });
```
Without success. But with AsInsert, it works(!). Here are the results:
```
INSERT INTO [table] ([column]) VALUES (@column)
UPDATE [table] SET [column] = 'SqlKata.UnsafeLiteral'
```

By the way, you should move the Attributes (ColumnAttribute, ...) into another namespace, so they will not conflict with System.ObjectModel.Annotations.Schema. And even Dapper tries to float the namespace with a new ColumnAttribute, if using Dapper.Contrib. So, there's a lot of conflict potential.

**Comment by ahmad-moussawi** [comment]
@arakis Thanks for reporting, just fixed this here is the relevant PR #355 
You can get the latest version from the nightly builds for now.

**Comment by ahmad-moussawi** [comment]
Available now on the latest package.
